<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionPrintf"/>
    <addaction name="actionQuit_Q"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="undoaction"/>
    <addaction name="redoaction"/>
    <addaction name="separator"/>
    <addaction name="copyaction"/>
    <addaction name="pasteaction"/>
    <addaction name="cutaction"/>
    <addaction name="separator"/>
    <addaction name="selectAllaction"/>
    <addaction name="separator"/>
    <addaction name="dateaction"/>
    <addaction name="separator"/>
    <addaction name="fontaction"/>
    <addaction name="coloraction"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="actionPrintf">
   <property name="text">
    <string>打印(&amp;P)</string>
   </property>
  </action>
  <action name="actionQuit_Q">
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
  </action>
  <action name="undoaction">
   <property name="text">
    <string>撤销(&amp;u)</string>
   </property>
  </action>
  <action name="redoaction">
   <property name="text">
    <string>重做(&amp;R)</string>
   </property>
  </action>
  <action name="copyaction">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="pasteaction">
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
  </action>
  <action name="cutaction">
   <property name="text">
    <string>剪切(&amp;X)</string>
   </property>
  </action>
  <action name="selectAllaction">
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
  </action>
  <action name="dateaction">
   <property name="text">
    <string>日期(&amp;D)</string>
   </property>
  </action>
  <action name="fontaction">
   <property name="text">
    <string>字体(&amp;F)</string>
   </property>
  </action>
  <action name="coloraction">
   <property name="text">
    <string>颜色(&amp;L)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
